src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/ifs/InterestForecastingRequests.scala

```diff
@@ -276,4 +276,21 @@ object InterestForecastingRequests extends ServicesConfiguration {
       .headers(requestHeaders)
       .body(StringBody(noInterestBearing))
       .check(status.is(200))
-}
+
+
+val interestType: String =s"""[
+                             |    {
+                             |        "mainTrans": "2000",
+                             |        "subTrans": "1000"
+                             |    }
+                             |]""".stripMargin
+
+
+  def debtInterestTypeRequest(baseUri: String): HttpRequestBuilder =
+    http("debt Interest Type")
+      .post(s"$baseUri/debt-interest-type")
+      .headers(requestHeaders)
+      .body(StringBody(interestType))
+      .check(status.is(200))
+
+  }
```

Comment:
Inconsistent indentation. It made the PR a little harder to read without an IDE. 

Should be easy to apply the diff straight from here, but I'm not clear on how you would verify that nothing broke.

```suggestion
  val interestType: String = s"""[
                                |    {
                                |        "mainTrans": "2000",
                                |        "subTrans": "1000"
                                |    }
                                |]""".stripMargin

  def debtInterestTypeRequest(baseUri: String): HttpRequestBuilder =
    http("debt Interest Type")
      .post(s"$baseUri/debt-interest-type")
      .headers(requestHeaders)
      .body(StringBody(interestType))
      .check(status.is(200))

}

```